scripts/docker/rhel/Dockerfile

```diff
@@ -0,0 +1,27 @@
+#
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+
+# Dockerfile that creates a container suitable to build dotnet-cli
+FROM microsoft/dotnet-buildtools-prereqs:rhel-7-rpmpkg-c982313-20174116044113
+
+# Install from sudo main package TODO This package needs to be mirrored
+RUN yum install -y https://www.sudo.ws/sudo/dist/packages/RHEL/7/sudo-1.8.20-3.el7.x86_64.rpm \
```

Comment:
@wli3 Only the host needs to be. The container picks up the registrations and the repositories from the host. If the host is not registered, neither the host nor the container gets access to subscriptions and updates.